arangod/Cluster/ClusterFeature.h

```diff
@@ -227,6 +231,13 @@ class ClusterFeature : public ArangodFeature {
   bool _unregisterOnShutdown = false;
   bool _enableCluster = false;
   bool _requirePersistedId = false;
+  // The following value indicates what HTTP code should be returned if
+  // a configured write concern cannot currently be fulfilled. The old
+  // behaviour (false, currently the default) means that a 403 FORBIDDEN
+  // with an error of 1004 COLLECTION READ ONLY is returned. The new
+  // behaviour (true) would be a HTTP 503 SERVER UNAVAILABLE with an error of
+  // 1429 ERROR_REPLICATION_WRITE_CONCERN_NOT_FULFILLED is returned.
```

Comment:
```suggestion
  // behavior (currently the default) means that a 403 FORBIDDEN
  // with an error of 1004 COLLECTION READ ONLY is returned. It is possible to
  // adjust the behavior so that a HTTP 503 SERVER UNAVAILABLE with an error of
  // 1429 ERROR_REPLICATION_WRITE_CONCERN_NOT_FULFILLED is returned.
```